content/en/platform/corda/4.11/community/api-flows.md

```diff
@@ -774,7 +774,26 @@ It’s possible to forcibly terminate the erroring finality handler using the `k
 
 {{< /warning >}}
 
+#### Two Phase Finality
 
+From Corda 4.11 on, the finality protocol has changed to improve resilience and recoverability.
```

Comment:
This reads like a release note?

Reply:
Rewrote these to omit "From Corda 4.11 on".